### ISSUE TITLE
Add Nestjs Module for typeDORM

### ISSUE THREAD

**Issue by whimzyLive** [typedorm/typedorm#20]
Add Drm module for root to be called by nestjs on initialization.

something like: https://github.com/chanlito/nestjs-extensions/tree/master/src/typeorm

**Comment by meeech** [comment]
hey, just adding a more recent link to the typeorm module. 
https://github.com/nestjs/typeorm
We're working with nestjs, and I was thinking to try out your project - I'll see what I can do about making a nice nestjs module

**Comment by whimzyLive** [comment]
@meeech 
Great! Contributions are always welcome. You can [TypeORM For NestJS](https://github.com/nestjs/typeorm) as the starting guide as most of the design decisions of TypeDORM are inspired by TypeORM.

**Comment by meeech** [comment]
yeah was looking through. pretty novice with nestjs still, so we'll see how far i get - i might just focus on an initial implementation to do the basics and see where it goes.

**Comment by tangentlin** [comment]
Definitely well worth the effort for integration, NestJS targets enterprise development, which shares similar goal of an ORM like TypeDORM.

**Comment by ChenNima** [comment]
Hey guys, if you are still looking for a NestJS TypeDorm integration, I've made one https://www.npmjs.com/package/@nest-dynamodb/typedorm following the guide from https://github.com/nest-modules/ioredis and https://www.npmjs.com/package/@nestjs/typeorm. I'm still working on CI and documentations, please feel free to try it out and fire issues and PRs.
